NTO/Automation/entities/ActionHandlerCapability.ttl

```diff
@@ -0,0 +1,32 @@
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix ogit.Automation:        <http://www.purl.org/ogit/Automation/> .
+@prefix ogit:                   <http://www.purl.org/ogit/> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit.Automation:ActionHandlerCapability
```

Comment:
I would call it only "Capability"

Reply:
or ActionCapability  ?  without "Action" it might to too general